File: swish_hackathon.py
from math import *
def main():
    ip=input()
    cost=int(ip.split(' ')[0])
    disc=int(ip.split(' ')[1])
    required=0
    while(cost>0):
        required=required+cost
        cost=floor(cost-(cost*(disc/100)))
    print(required)
main()

File: swish_hackathon.c
#include<stdio.h>
#include<math.h>
int main(){
    
    float cost,disc;
    int required=0;
    scanf("%f%f",&cost,&disc);
    while(cost>0){
        required+=cost;
        cost=floor(cost-(cost*(disc/100)));
    }
    printf("%d\n",(int)required);
    return 0;
    
}